pkg/lockfile/types.go

```diff
@@ -1,15 +1,13 @@
 package lockfile
 
-import "github.com/google/osv-scanner/pkg/models"
-
 type PackageDetails struct {
 	Name       string    `json:"name"`
 	Version    string    `json:"version"`
 	Commit     string    `json:"commit,omitempty"`
 	Ecosystem  Ecosystem `json:"ecosystem,omitempty"`
 	CompareAs  Ecosystem `json:"compareAs,omitempty"`
-	Start      models.FilePosition
-	End        models.FilePosition
```

Comment:
I think we still need a structure to handle this as we will soon work on adding the columns back (CI team is asking for this now). I think the LinePosition structure would be ideal